api/commands/misc/timezone.js

```diff
@@ -0,0 +1,44 @@
+const { Command } = require('discord.js-commando');
+const bot         = require('../../bot.js');
+const exceptions  = require('../../util/exceptions.json');
+const selectTz    = require('../../../db/queries/selectTimezone.js');
+const insertTz    = require('../../../db/queries/insertTimezone.js');
+const moment      = require('moment-timezone');
+
+const command = {
+  name: 'timezone',
+  group: 'misc',
+  memberName: 'timezone',
+  description: 'Set your timezone. To view a list of zones, please visit ' +
+    '`https://en.wikipedia.org/wiki/List_of_tz_database_time_zones`',
+  examples: ['rbot timezone America/Los_Angeles'],
+  args: [
+    {
+      key: 'content',
+      prompt: 'What is your timezone?',
+      type: 'string'
+    }
+  ]
+};
+
+module.exports = class TimezoneCommand extends Command {
+  constructor(client) {
+    super(client, command);
+  }
+
+  run(msg, { content }) {
+    let username = msg.message.author.username;

```

Comment:
dont think you need to set these since you use them only once on line 37. can probably just put both of these vars there

Reply:
I see your point.

Originally, I did have inline-variables but the code was looking butt-ugly that way.

That is, `return insertTz([msg.message.author.username, msg.message.author.discriminator, content])` looked horrid.

Reply:
would see how something like this looks:
```
return insertTz([
  msg.message.author.username,
  msg.message.author.discriminator,
  content
])
```
Otherwise it's scoped variables that will get gc afterwards so it's no big deal